nais/app/labs.yaml

```diff
@@ -41,3 +41,5 @@ spec:
       value: "false"
     - name: ASSET_PREFIX
       value: "https://cdn.nav.no/flex/spinnsyn-frontend"
+    - name: FEILMELDINGER_URL
+      value: "https://www.nav.no/syk/feilmeldinger"
```

Comment:
dette er bare i labs, mangler URL for prod.

Kunne eventuelt bare brukt relativ url ?

Reply:
Lagt til i prod.yaml.